frontend/src/pages/QueryData/QueryData.tsx

```diff
@@ -113,14 +125,17 @@ class QueryData extends React.Component<QueryDataProps> {
   }
 
   private exportToCSV = () => {
+    this.setState({ exporting: true });
     const data = this.props.page.getIn([ 'data', 'results' ]) as List<OperationDataMap>;
     if (data.count()) {
-      const csv = encodeURI(`data:text/csv;charset=utf-8,${unparse(data.toJS())}`);
       const filename = 'export.csv';
       const link = document.createElement('a');
-      link.setAttribute('href', csv);
+      const csvData = new Blob([ unparse(data.toJS()) ], { type: 'text/csv' });
+      const url = URL.createObjectURL(csvData);
```

Comment:
@edwinmp is it possible to add a UTF8 BOM to the beginning of this file? Excel cannot natively handle UTF8 without it.

Should be something like 
```
\ufeff
```
Via https://stackoverflow.com/a/17879474

Reply:
Done!